src/main/java/org/togetherjava/tjbot/commands/example/CommandExample.java

```diff
@@ -0,0 +1,92 @@
+package org.togetherjava.tjbot.commands.example;
+
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.CommandData;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.requests.restaction.CommandCreateAction;
+import org.togetherjava.tjbot.commands.ICommand;
+
+/**
+ * Example command
+ *
+ */
+public class CommandExample implements ICommand {
+
+    /**
+     * The command name is example, so it returns example.
+     *
+     * @return
+     *  {@link String} with the command's name
+     */
+    @Override
+    public String getCommandName() {
+        return "example";
+    }
+
+    /**
+     * The description
+     *
+     * @return
+     *  A {@link String} with the description
+     */
+    @Override
+    public String getDescription() {
+        return "Example command! Allows you to spam ping someone!";
+    }
+
+    /**
+     * You can make commands guild only by overriding this method.
+     * <br> You can only ping people in a guild, so this command is guild only.
+     *
+     * @return
+     *  {@link Boolean} of {@code true}
+     */
+    @Override
+    public boolean isGuildOnly() {
+        return true;
+    }
+
+    /**
+     * Adds the "User" and "Amount of pings" option.
+     * <br> This is for the GUI, so the user sees what options it has.
+     *
+     * @param commandData
+     *  The thing we will add our options to!
+     * @return
+     *  A {@link CommandCreateAction} with the required options for this command
+     */
+    @Override
+    public CommandData addOptions(CommandData commandData) {
+        return commandData.addOptions(
+                // * add the first option
+                new OptionData(OptionType.USER, "User", "User that will be pinged!", true),
```

Comment:
I think option names have to be all lower case. I don't quite remember.

Reply:
Could be, command names and the description also have some limits so this will have to be added to the docs.
But didn't do that currently since it's more of a draft.

Reply:
Fixed it and added requirements section to 2 ICommand methods